stacks-node/src/tests/nakamoto_integrations.rs

```diff
@@ -14929,3 +14931,394 @@ fn contract_limit_percentage_mempool_strategy_low_limit() {
 
     run_loop_thread.join().unwrap();
 }
+
+#[test]
+#[ignore]
```

Comment:
You'll need to add this test to the `bitcoin-tests.yml` workflow file so it runs in CI